test/sharness/t0040-add-and-cat.sh

```diff
@@ -285,6 +285,23 @@ test_expect_success "'ipfs add -r' output looks good" '
 	test_cmp expected actual
 '
 
+test_expect_success "'ipfs add -rn' succeeds" '
+	mkdir mountdir/moons &&
+	echo "Hello Europa!" >mountdir/moons/europa.txt &&
+	echo "Hello Titan!" >mountdir/moons/titan.txt &&
```

Comment:
I'd like to see another deeper directory tested here, just to be sure that we don't just have the root dir special cased for printing.